cgpsa.conf

```diff
@@ -574,23 +574,62 @@
 #
 # use_dma_spam_mailbox = false
 
-# The name of the mailbox to use for direct mailbox addressing when 
-# the "use_dma_spam_mailbox" setting is enabled. 
+# The name of the mailbox to use for direct mailbox addressing when
+# the "use_dma_spam_mailbox" setting is enabled.
 #
 # spam_mailbox_name = SPAM
 
 # A flag that determines whether CGPSA will discard spam messages above
 # a certain threshold (set in "discard_threshold"). This causes all
 # messages rated above the threshold level to be discarded without a
-# delivery attempt. 
+# delivery attempt.
 #
 # auto_discard = false
 
-# A threshold score above which CGPSA will discard messages if 
+# A threshold score above which CGPSA will discard messages if
 # "auto_discard" is enabled. For instance, with the default setting, any
 # message with a SpamAssassin score >= 25.0 will be discarded with no
 # delivery attempt if "auto_discard" is enabled.
 #
 # discard_threshold = 25.0
 
+# A flag that determines whether CGPSA adds an "X-Junk-Score" header
+# (see the "Junk Processing Simplified Rules" in the CGP manual). Th
+# header has a score generated based on the thresholds in the
+# "x_junk_score_thresholds" setting below. The default is false,
+# because it is possible that another filter that adds the "X-Junk-Score"
+# header may be used in conjunction wih CGPSA.
+#
+# x_junk_score_header = false
+
+# The thresholds for the "X-Junk-Score" header. These are six
+# space-separated numbers, each of which is a SpamAssassin score at
+# which the next "X" in an "X-Junk-Score: SCORE [XXX]" header is added,
+# where SCORE is a constant based on the number of Xs the email is given
+# and the "official" ranges for "X-Junk-Score" given at
+# https://www.communigate.ru/CGPKAS/ for the CGP Kaspersky plugin.
+#
+# For example, with the default settings, the following SpamAssassin
+# scores would result in the following headers:
+#
+# 0.0 - X-Junk-Score: 0 []
+# 1.0 - X-Junk-Score: 20 [X]
+# 4.0 - X-Junk-Score: 60 [XX]
+# 8.0 - X-Junk-Score: 85 [XXX]
+# 9.0 - X-Junk-Score: 93 [XXXX]
+# 9.5 - X-Junk-Score: 98 [XXXXX]
+# 10.0 - X-Junk-Score: 100 [XXXXXX]
+#
+# Any score below the first threshold, or above the last, will have a
+# score of 0 or 100, respectively. If the scores in the list are not
+# floating point numbers (including a decimal point), or if any score
+# in the list is less than the previous one, the setting will be
+# ignored. It is legal to have identical values in the list, in which
+# case some X counts will be skipped (for example, if the second value
+# was changed to 8.0, then everything with a spam score above 0 and
+# below 8.0 would get a single X, everything above 8.0 and below
+# 9.0 would get 3 Xs, and nothing would ever get 2 Xs).
+#
+# x_junk_score_thresholds = 0.00001 4.0 8.0 9.0 9.5 10.0
```

Comment:
Does `0.00001` correspond to `1.0 - X-Junk-Score: 20 [X]` documented above? Maybe this should be more clear. How about setting both to `0.1` ?

Reply:
I do think changing it in the documentation and defaults may make sense, though.

Reply:
Can be resolved.